src/components/Footer/Footer.less

```
@@ -0,0 +1,198 @@
+@import (reference) url('@cfpb/cfpb-design-system/src/cfpb-design-system.less');
```

IIRC these imports can be simplified as:

```suggestion
@import (reference) '@cfpb/cfpb-design-system';
```

---

It does work but, because it's a directory, the shortened version fails in terms of being easily navigable (CMD + click) when trying to explore the design system content.